api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -27,8 +33,8 @@ message ExtAuthz {
 
   // The filter's behaviour in case the external authorization service does
   // not respond back. If set to true then in case of failure to get a
```

Comment:
I know it's a bit too long and wordy. How about:
*"When it is set to true then in case of failure to communicate with the authorization service traffic will be permitted."*
Any suggestions on how to better phrase it?

Reply:
Thanks! That's clearer. Suggestion:
`When it is set to true, Envoy will also allow traffic in case of communication failure between the authorization service and the proxy.`

Reply:
only on failure. If it's a explicit Deny response then that is NOT overridden by this flag.

Reply:
Awesome! Thanks!